src/dns.rs

```diff
@@ -164,10 +164,7 @@ mod test {
     #[test]
     fn resolver() {
         // Add simple logger, if not possible, that's fine too
-        match SimpleLogger::new().init() {
-            Ok(_) => {}
-            Err(_) => {}
-        }
+        let _ = SimpleLogger::new().init();
```

Comment:
When i think about this, I'm not sure if we want to to this here. It maybe would make sense to use the `SimpleLogger` instead of our own and invoke the logger init somewhere in the main fn. The other option I'd see, to upgrade our logger and use this instead. I guess it makes sense to only have one logger.